src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -115,6 +118,50 @@ class RemoteMachineTrainingService implements TrainingService {
         }
         this.log.info('Remote machine training service exit.');
     }
+    
+    /**
+     * give trial a ssh connection
+     * @param trial 
+     */
+    public async scheduleTrialSSHClient(trial: RemoteMachineTrialJobDetail): Promise<void> {
```

Comment:
Suggest to rename the function to 'allocateSSHClientForTrial'

Reply:
You didn't use await in your function. So no need to mark the function as async and return Promise

Reply:
changed to `allocateSSHClientForTrial`